lib/ansible/plugins/netconf/sros.py

```diff
@@ -19,6 +19,23 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+DOCUMENTATION = """
+---
+netconf: sros
```

Comment:
Based on the feedback from the Ansible team (Andrius, ...) we've taken the effort building a Nokia SR OS collection that contains terminal, cliconf and netconf plugins. You might want to set this SROS plugin to "deprecated" while telling people to use collection "nokia.sros" instead.

Please provide guidance, how to change the existing plugin on galaxy, to add the device handler reference!

Motivation:
Using collections on Galaxy allows Nokia to update the content more frequently while distributing changes (fixes/improvements, new features) to the community outside regular Ansible release cycles. Also we do not rely anymore on the God's mercy that changes contributed by us get merged back into Ansible.

Also the message we've received is saying, that in the future all vendor specific code will be removed from Ansible core and need to be moved to Galaxy anyhow.

Reply:
Most of the content is planned to be cut out from the ansible core before the next ansible release. After the cut out happens the only location sros plugin will reside is in `nokia.sros` collection, so no need to mark it as depcreated. Since we are in the transition phase it needs to be maintained at both places for some time until the content cutout happens. 

As per your comment, I will change the value from `alu` to `default` in this PR

In your collection update the `DOCUMENTATION` section in netconf plugin similar to what is added in this PR and test it after applying this PR in your environment.

Reply:
Sure will do. Thanks for your support on this.

Reply:
Onm a side note looking at the `nokia.sros` collection it make sense to also move sros related content in `modules` and `module_utils` folder into `nokia.sros` collection

Reply:
You can use this content migration tool https://github.com/ansible-community/collection_migration (in case you have not used it) to generate the `nokia.sros` collection.

Reply:
Not sure, what other content is needed to be migrated. The SROS plugins make the existing SROS modules obsolete, while people now can use the standard CLI and NETCONF modules. In that sense, I would also claim that `module_utils` are no longer needed.

About `actions` I need to validate, what is still left. At least I am not aware of anything.

Reply:
At least in my peer group, everybody is happy with the current content. I might add another Nokia SR OS module, just to have something available. Just need to think about a good use-case here.